sdks/python/src/opik/integrations/dspy/utils.py

```diff
@@ -0,0 +1,88 @@
+from typing import Any, Dict, List
+import dspy
+import string
+
+STYLES = """
+classDef ReAct fill:#90EE90
+classDef Predict fill:#F08080
+classDef ChainOfThought fill:#ADD8E6
+classDef Tools fill:##D3D3D3
+"""
+
+
+def get_mermaid_graph(module: dspy.Module) -> str:
```

Comment:
This is the only public function in this module, the rest should have underscore before their names.
Let's also call it `build_mermaid_graph_from_module`